RealmSwift-swift2.0/List.swift

```diff
@@ -69,7 +69,6 @@ public final class List<T: Object>: ListBase {
 
     /// Creates a `List` that holds objects of type `T`.
     public override init() {
-        // FIXME: use T.className()
         super.init(array: RLMArray(objectClassName: (T.self as Object.Type).className()))
```

Comment:
I had originally written this FIXME because I didn't understand why the `(T.self as Object.Type).className()` gymnastics were necessary. Now I understand it's because it's ambiguous (NSObject on OS X has className too). So yes, the FIXME is no longer necessary.